### ISSUE TITLE
Should Canon CR3 format explicitly not be detected as MP4?

### ISSUE THREAD

**Issue by evilpie** [whatwg/mimesniff#149]
> Canon has introduced a new RAW image format CR3 which is based on the iso mp4 container

https://gitlab.freedesktop.org/xdg/shared-mime-info/-/issues/157
https://github.com/lclevy/canon_cr3/blob/master/readme.md
https://bugzilla.mozilla.org/show_bug.cgi?id=1725190

**Comment by GPHemsley** [comment]
I don't think we want to be sniffing the CR3 format specifically, either positively or negatively.

However, [a later comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1725190#c5) in the linked bug indicates that Firefox is using the `isom` brand to detect it as MP4.

Since that brand is not part of the MP4 sniffing algorithm, the question on the table is: should the MP4 sniffing algorithm be changed to include the `isom` brand, or should Firefox be changed to remove it?
